tests/parsers/winreg_plugins/test_lib.py

```diff
@@ -96,7 +96,7 @@ def _ParseKeyWithPlugin(
       registry_key (dfwinreg.WinRegistryKey): Windows Registry Key.
       plugin (WindowsRegistryPlugin): Windows Registry plugin.
       file_entry (Optional[dfvfs.FileEntry]): file entry.
-      knowledge_base_values (Optional[dict[str, str]}): knowledge base values.
+      knowledge_base_values (Optional[dict(str, str)]): knowledge base values.
```

Comment:
`dict(str, str)` why this change?

if this is a deliberate change then it is unclear to me when `[]` is used and `()` is used please add this to the style guide or make it consistently `[]`

Reply:
Done